components/builder-originsrv/src/migrations/2018-03-05-224023_origin_secrets/up.sql

```diff
@@ -0,0 +1,53 @@
+CREATE SEQUENCE IF NOT EXISTS origin_secrets_id_seq;
+
+CREATE TABLE IF NOT EXISTS origin_secrets (
+  id bigint PRIMARY KEY DEFAULT next_id_v1('origin_secrets_id_seq'),
+  origin_id bigint REFERENCES origins(id),
+  owner_id bigint,
+  name varchar(256),
+  value text,
+  created_at timestamptz DEFAULT now(),
+  updated_at timestamptz DEFAULT now(),
+  UNIQUE (origin_id, name)
+);
+
+CREATE OR REPLACE FUNCTION insert_origin_secret_v1 (
+  os_origin_id bigint,
+  os_name varchar(256),
+  os_value text
+) RETURNS SETOF origin_secrets AS $$
+    BEGIN
+      RETURN QUERY INSERT INTO origin_secrets (origin_id, name, value)
+          VALUES (os_origin_id, os_name, os_value)
+          RETURNING *;
+      RETURN;
+    END
+$$ LANGUAGE plpgsql VOLATILE;
```

Comment:
We wanted to standardize on SQL instead of plpgsql where possible - consider switching it over.